### PR TITLE
VZ:4796: update install for POKO 

### PR DESCRIPTION
**Pull request by bobdonat**
# Description

This pull request updates the install to include new POKO resources.  In addition, a couple of problems were fixed in the metrics binding pod labeler.

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by davisgomes — DISMISSED** [review]
Looks good. I will copy over some of the log message changes to my webhook updates.

File: application-operator/controllers/webhooks/metrics-binding-labeler-pod.go [hunk 1]
```diff
@@ -60,12 +59,20 @@ func (a *LabelerPodWebhook) handlePodResource(req admission.Request, log *zap.Su
...
 	if len(pod.OwnerReferences) != 0 {
-		workloads, err := a.getWorkloadResource(nil, req.Namespace, pod.OwnerReferences, log)
+		workloads, err := a.getWorkloadResource(nil, req.Namespace, pod.OwnerReferences, a.RestMapper, log)
 		if err != nil {
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
+		for _, workload := range workloads {
+			// If we have an owner ref that is an OAM ApplicationConfiguration resource then we don't want
+			// to label the pod to have the app.verrazzano.io/workload label
+			if workload.GetKind() == "ApplicationConfiguration" && workload.GetAPIVersion() == "core.oam.dev/v1alpha2" {
```
Comment (kminder) [inline application-operator/controllers/webhooks/metrics-binding-labeler-pod.go:72]: Hmm.  Probably don't want to make this version specific.

File: application-operator/controllers/webhooks/metrics-binding-labeler-pod.go [hunk 2]
```diff
@@ -95,16 +102,17 @@ func (a *LabelerPodWebhook) handlePodResource(req admission.Request, log *zap.Su
...
 	for _, ownerRef := range ownerRefs {
+		// Find preferred GroupVersionResource
 		group, version := controllers.ConvertAPIVersionToGroupAndVersion(ownerRef.APIVersion)
-		resource := schema.GroupVersionResource{
-			Group:    group,
-			Version:  version,
-			Resource: pluralize.NewClient().Plural(strings.ToLower(ownerRef.Kind)),
+		mapping, err := mapper.RESTMapping(schema.GroupKind{Group: group, Kind: ownerRef.Kind}, version)
+		if err != nil {
+			log.Errorf("Failed getting resource mapping: %v", err)
```
Comment (kminder) [inline application-operator/controllers/webhooks/metrics-binding-labeler-pod.go:111]: Paul was ok with a %v?

Comment (bobdonat) [reply application-operator/controllers/webhooks/metrics-binding-labeler-pod.go:111]: He did not say anything about it.  I did a search in platform operator and it is used often.

File: application-operator/controllers/webhooks/metrics-binding-labeler-pod.go (hunk 1 above)
Comment (bobdonat) [reply application-operator/controllers/webhooks/metrics-binding-labeler-pod.go:72]: So only check Kind and Group?

File: application-operator/controllers/webhooks/metrics-binding-labeler-pod.go (hunk 2 above)
Comment (davisgomes) [reply application-operator/controllers/webhooks/metrics-binding-labeler-pod.go:111]: Yes he wanted %v to get the full error stack trace.

**Review by kminder — APPROVED** [review]
Approved.

File: application-operator/controllers/webhooks/metrics-binding-labeler-pod.go (hunk 1 above)
Comment (bobdonat) [reply application-operator/controllers/webhooks/metrics-binding-labeler-pod.go]: fixed

**Pull request merged**